Assets/UI/actionpanel.lua

```diff
@@ -1079,44 +1080,52 @@ end
 
 -- ===========================================================================
 function OnLocalPlayerTurnBegin()
-    -- Standard disable is set to false in the refresh.
-    -- This extra level of input catching is done when tutorial has raised
-    -- this boolean and will prevent spam clicking through; as the tutorial
-    -- system itself sets ENABLED as it's hiding all controls.
-    if m_isSlowTurnEnable then
-        Controls.TutorialSlowTurnEnableAnim:SetHide(false);
-        Controls.TutorialSlowTurnEnableAnim:SetToBeginning();
-        Controls.TutorialSlowTurnEnableAnim:RegisterEndCallback(
-            function()
-                Controls.TutorialSlowTurnEnableAnim:SetHide(true);
-            end
-        );
-        Controls.TutorialSlowTurnEnableAnim:Play();
-    end
-    ContextPtr:RequestRefresh();
+    -- If the player is dead (observing), then automatically end their turn
```

Comment:
the comparer is not doing well with this comparision
everything from line 1082 to line 1104 on the LEFT side matches everything from line 1088 to line 1110 on the RIGHT side.
all that code that previously made up all of the OnLocalPlayerTurnBegin function is now in the "else" case below